src/jdk.crypto.ec/share/classes/sun/security/ec/SunEC.java

```diff
@@ -161,7 +174,24 @@ public Object newInstance(Object ctrParamObj)
                     }
                 } else  if (type.equals("KeyPairGenerator")) {
                     if (algo.equals("EC")) {
-                        return new ECKeyPairGenerator();
+                        /* Disabling OpenSSL usage in AIX due to perfomance regression observed */
+                        if (useNativeECKeyGen &&
+                           (NativeCrypto.getVersionIfAvailable() >= 0x10100000) &&
+                           (!System.getProperty("os.name").contains("AIX"))) {
+                            if (nativeCryptTrace) {
+                                System.err.println("EC KeyPair Generation - Using OpenSSL integration.");
+                            }
+                            return new NativeECKeyPairGenerator();
+                        } else {
+                            if (nativeCryptTrace) {
+                                if (System.getProperty("os.name").contains("AIX")) {
+                                    System.err.println("EC KeyPair Generation - Not using OpenSSL integration in AIX.");
+                                } else if (NativeCrypto.getVersionIfAvailable() < 0x10100000) {
+                                    System.err.println("EC KeyPair Generation - Not using OpenSSL integration due to older version of OpenSSL (<1.1.0).");
```

Comment:
Please use the named constant `NativeCrypto.OPENSSL_VERSION_1_1_0`.

Reply:
Changes version check to use constant.